Post for customers

probabri opened the pull request:
Wrote post api for customers. 
<img width="719" alt="image" src="https://user-images.githubusercontent.com/76490401/159571224-37cbc46f-37e1-4a75-8bd9-7b972d74c024.png">

BaltejToorTJX reviewed (changes requested):
Do we want to send a status code on successful calls as well? Checking to make sure so I can adjust moving forward. Also, let's change to using 404 for the error call in addition to passing the error instead of the generic message (unless there is a rationale for that that I'm unaware of).

You could also remove allowNull completely from customerNotes since the default behaviour is true anyways.

probabri commented:
So I updated the status codes - we probably don't want to do a 404 (we can discuss) and added success ones as appropriate. 
Removed allow null completely from the customer note model - good call

danieltrying reviewed (approved):
LGTM
